### DISCUSSION TITLE
Pairing e comunicazione zigbee non funzionante

### DISCUSSION THREAD

**Discussion by sanzari88** [indomus/forum#3107] [HUB: Home Assistant]
Salve a tutti sto implementando una configurazione domotica basata su Raspeberry PI 3 con immagine HASSOS.
A bordo ho installato il broker e client MQ e zigbee2Mq per ricevere i messaggi zigbee utilizzando come antenna ZBDONGLE-E.
Attualmente sto provando con i seguenti componenti 
- AUBESS Mini Smart Switch
- Mini Smart Switch iHSW02

Dopo essere riuscito ad associare i componenti la comunicazione fra la dashboard e loro non va e dopo alcuni secondi che provo a far commutare i rele mi esce nei log il seguente errore.

```
{"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Delivery failed for {"profileId":260,"clusterId":6,"sourceEndpoint":1,"destinationEndpoint":1,"options":4160,"groupId":0,"sequence":228})'
[2024-06-03 23:09:55] error: 	z2m: Publish 'set' 'state' to 'Led Salone2' failed: 'Error: ZCL command 0xa4c1389c213c9b8a/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Delivery failed for {"profileId":260,"clusterId":6,"sourceEndpoint":1,"destinationEndpoint":1,"options":4160,"groupId":0,"sequence":229})'
[2024-06-03 23:09:59] error: 	z2m: Publish 'set' 'state' to 'Led Salone2' failed: 'Error: ZCL command 0xa4c1389c213c9b8a/1 genOnOff.on({}, 
```

Mentre per il dispositivo in foto, uno WETEN, non riesco proprio ad effettuare il pairing.

![image](https://github.com/indomus/forum/assets/32314131/0a9927b1-c059-411e-bd8b-b723c936d311)

Qualcuno ha avuto lo stesso problema o può indirizzarmi verso qualche verifica più precisa che posso effettuare?
Grazie

**Comment by Scialla** [comment]
L'errore è un timeout, prova a cancellare il dispositivo e rifare il pairing

quello della foto mi sa che non è compatibile:
https://www.zigbee2mqtt.io/supported-devices/#v=Weten

Dovresti provare con:
https://www.zigbee2mqtt.io/guide/configuration/more-config-options.html#external-converters

o apri una issue con richiesta di aggiunta nei device supportati

**Reply by sanzari88** [reply]
Ho provato più di una volta a rifare il pairing ma nulla.. ho anche provato a fare una nuova installazione di tutto...

Il venditore mi dice che è compatibile, e cercandolo sul sito zigbee2mqtt lo trovo (è comunque un tuya) anche se marcato diversamente
https://www.zigbee2mqtt.io/devices/TS0002_switch_module_4.html

A livello di configurazione di zigbee2mqtt ci potrebbe essere qualche setting che posso migliorare? io ho lasciato la configurazione base, settando solo i seguenti parametri

`
data_path: /config/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: "-d -d"
  log: false
  homeassistant: true
mqtt:
  server: mqtt://core-mosquitto:1883
  user: mqtt_user
  password: mqtt_psw
serial:
  port: /dev/ttyACM0
  adapter: ember
advanced:
  channel: 11
`
mentre la ricerca dei device la faccio dal coordinatore

![image](https://github.com/indomus/forum/assets/32314131/97bf1e53-62b0-4d3e-b8e6-f51a2bc98866)

**Reply by KC-inDomus** [reply]
secondo me monta una versione firmware vecchia e non supportata (oppure troppo nuova)
